tests/testsuite/cargo_remove/multiple_dev/out/Cargo.toml

```diff
@@ -17,4 +17,4 @@ toml = "0.1"
 clippy = "0.4"
 
 [features]
-std = ["semver/std"]
+std = [ "semver/std"]
```

Comment:
This isn't addressed. Should we?

Reply:
This is the cost of this change and the new policy.  The user had a space before the entry, so we preserved that.

Every once in a while, I'm tempted to change `toml_edit` so it detects "default" formatting and preserves that notion so that when an edit happens, it continues to use the default.

Reply:
Let me know when we get it in toml_edit.